solr/solr-ref-guide/modules/deployment-guide/pages/metrics-reporting.adoc

```diff
@@ -231,6 +231,40 @@ This is only valid for the `ExponentiallyDecayingReservoir`.
 +
 The window size, in seconds, and only valid for the `SlidingTimeWindowReservoir`.
 
+`<maxHistogram>`::
```

Comment:
this is some hard-core docs... not sure 1% of readers will understand it.  Not necessarily a problem; you put thought into this.